src/pages/settings/advanced/advanced.html

```diff
@@ -16,6 +16,8 @@
         <div *ngIf="!isCopay" translate>However, unconfirmed funds are not allowed for spending with merchants, BitPay Card loads, or BitPay in-app gift card purchases.</div>
       </div>
     </label-tip>
+    <button ion-item class="with-label no-border" (click)="openWalletRecoveryPage()">{{'Recover old wallets' | translate}}</button>
```

Comment:
This need a condition that checks if profile old exist since we have plans to delete the old profile in the future and we don't want to continue showing this option

Reply:
Or maybe at the time, we remove it we can remove the code too. So maybe is ok this way. What do you think ?